### PR TITLE
YALB-256: Setup Lando

### PR DESCRIPTION
**Pull request by nJim**
### Description of work
- Creates a lando config file for all sites using the upstream. This uses the Pantheon recipe to leverage supporing commands such to sync Pantheon environments with the local dev.
- Includes an example local lando file to help developers set site specific environmental variables.
- Adds the local lando file to gitignore
- Includes minimal changes to the README as a future ticket includes deeper contribution instructions.

### Functional testing steps:
- [ ] Clone this repo: `git clone https://github.com/yalesites-org/yalesites-upstream-base.git`
- [ ] Checkout this branch `git checkout YALB-256-lando`
- [ ] Make a file for local lando settings in the repo root named `.lando.local.yml` with the contents:
```
name: yalesites4kupstreamtest16
config:
  id: ed6c6a92-74fe-46f2-9bc5-b500b2ecc0b7
  site: yalesites4kupstreamtest16
```
- [ ] Run `lando start` to initialize the environment
- [ ] Run `lando pull --code=none --database=dev --files=none` 
- [ ] Visit the site and verify Drupal is installed. [https://yalesites4kupstreamtest16.lndo.site](https://yalesites4kupstreamtest16.lndo.site)
- [ ] Verify the site title reads "YaleSites Platform".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +31,21 @@ Files and directories important to the structure of the upstream include:
...
+### Requirements
+
+* [Composer](https://getcomposer.org/download/): PHP package manager. Version 2.x.
+* [Terminus](https://pantheon.io/docs/terminus): Pantheon CLI.
+* [Docker](https://docs.docker.com/install): Container system for virtualization.
+* [Lando](https://docs.lando.dev/basics/installation.html#system-requirements): Recipes and tools for Docker.
+
+### Setup
+* [Lando CA](https://docs.devwithlando.io/config/security.html): Optional certificate authority setup.
+* Public key for your machine uploaded to Pantheon.
```
Comment (dens0ne) [inline README.md:48]: If we have adding the public key as a setup step, we might as well suggest developers to create a machine token for using Terminus.

File: .gitignore [hunk 2]
```diff
@@ -95,3 +95,6 @@
 
 # Ignore .env files as they are personal
 #/.env
+
+# Ingnore local lando settings. Each site will have unique values.
```
Comment (dens0ne) [inline .gitignore:99]: Typo in word "ignore"

**Review by dens0ne — CHANGES_REQUESTED** [review]
This looks good! I would just like to have a small typo fixed and your thoughts on a small note in the setup steps.

File: .gitignore (hunk 2 above)
Comment (nJim) [reply .gitignore]: TY. Fixed.

**Review by dens0ne — APPROVED** [review]
:shipit:

**Pull request merged**